vdr/cmd/cmd.go

```diff
@@ -64,12 +64,26 @@ func Cmd() *cobra.Command {
 }
 
 func createCmd() *cobra.Command {
-	return &cobra.Command{
+	// needs to be initialized for pflags, values will be overwritten with defaults from pflag
+	trueVal := true
+	falseVal := false
+	var sliceVar []string
+	var createRequest = api.DIDCreateRequest{
```

Comment:
does pflag assign a new ptr bool var or does it just change the value? If it's the last, if the last `falseVal` is false, all are false? (and vice versa for `trueVal`). Because you pass the same var for many properties.

Reply:
changed to `new(bool)`